src/ui/shared/requests/getPermissionsWithWallets.ts

```diff
@@ -1,4 +1,4 @@
-import { ethers } from 'ethers';
+import { getAddress } from 'ethers/lib.esm/address';
```

Comment:
https://github.com/zeriontech/zerion-wallet-extension/pull/682/files#r1897848519

I can see 2 types of imports. Is it a big difference?
Current one doesn't look like an expected way to import functions :)

Reply:
Let me update to import directly from ethers